From ecec9a7eaaf364cf2623c3c447e6ef5913283b10 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jonas=20=C3=85dahl?= <jadahl@gmail.com>
Date: Fri, 13 Sep 2019 09:45:11 +0200
Subject: [PATCH 07/43] cogl/egl: Just notify about failure to create high
 priority context

Warning about it causes test case failures if there are not enough
privileges, which is likely when running the test suite locally.

https://gitlab.gnome.org/GNOME/mutter/merge_requests/797
---
 cogl/cogl/winsys/cogl-winsys-egl.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cogl/cogl/winsys/cogl-winsys-egl.c b/cogl/cogl/winsys/cogl-winsys-egl.c
index a6a0dca36..e48b65498 100644
--- a/cogl/cogl/winsys/cogl-winsys-egl.c
+++ b/cogl/cogl/winsys/cogl-winsys-egl.c
@@ -406,7 +406,7 @@ try_create_context (CoglDisplay *display,
                        &value);
 
       if (value != EGL_CONTEXT_PRIORITY_HIGH_IMG)
-        g_warning ("Failed to obtain high priority context");
+        g_message ("Failed to obtain high priority context");
     }
 
   if (egl_renderer->platform_vtable->context_created &&
-- 
2.23.0

